packages/UI/.env

```diff
@@ -1 +1 @@
-VITE_MNEMONIC="<your_mnemonic_here>"
\ No newline at end of file
+# VITE_MNEMONIC="<your_mnemonic_here>"
```

Comment:
I see this env var is used in component, why should we comment it ?

Reply:
As outlined in the documentation, there are two provider options available:

1. **Environment Variable**: Users can pass the `mnemonic` as an environment variable, and the account will be loaded accordingly.

2. **Threefold Connector Extension**: Alternatively, users can comment the line `VITE_MNEMONIC="<your_mnemonic_here>"` in the `.env` file, following the documentation instructions. This step is essential for utilizing the `Threefold Connector Extension`.

It's crucial to note that, when we publish the package on npm, users won't be able to directly pass their mnemonic. Therefore, the recommended approach is to comment out the `VITE_MNEMONIC` variable in the `.env` file, effectively unlocking the functionality provided by the `Threefold Connector Extension` for users.